repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -252,6 +257,67 @@
         </xs:simpleContent>
     </xs:complexType>
 
+    <xs:complexType name="ViaConfiguration">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    Contains the information needed to configure Repose's Via header.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:simpleContent>
+            <xs:extension base="papic:EmptyString">
+                <xs:attribute name="prefix" type="xs:string" use="optional">
+                    <xs:annotation>
+                        <xs:documentation>
+                            <html:p>
+                                An identifying string that will be returned in the Via header.
+                                If undefined and the repose-version is false, then no Via header will be added.
+                            </html:p>
+                        </xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="repose-version" type="xs:boolean" use="optional" default="true">
+                    <xs:annotation>
+                        <xs:documentation>
+                            <html:p>
+                                Indicates whether or not the Via header is created.
```

Comment:
It seems like this actually indicates whether or not to include the Repose version in the Via header.